roles/gather/README.md

```diff
@@ -32,14 +32,295 @@ Below is an example playbook demonstrating how to use the `gather` role, where w
       ansible.builtin.include_role:
         name: network.reports.gather
       vars:
+        format: json
         resources:
           - "bgp_global"
           - "bgp_address_family"
 ```
 
 Example Output
-When the playbook is executed successfully, the output will display the all_gathered_facts for the specified resources.
-
+When the playbook is executed successfully, the output will display the all_gathered_resources for the specified resources.
+- Yaml Output
+```yaml
+   all_gathered_resources:
+        bgp_address_family:
+            address_family:
+            -   afi: ipv4
+                networks:
+                -   prefix: 10.0.0.0/24
+                -   prefix: 172.16.0.0/16
+                safi: unicast
+            -   afi: ipv6
+                networks:
+                -   prefix: 2001:db8::/32
+                -   prefix: 2001:db8:abcd::/48
+                safi: unicast
+            as_number: '65001'
+        bgp_global:
+            as_number: '65001'
+            neighbors:
+            -   neighbor_address: 192.168.1.2
+                remote_as: '65002'
+                update_source: loopback0
+            -   neighbor_address: 192.168.2.1
+                remote_as: '65002'
+            -   neighbor_address: 2001:db8::2
+                remote_as: '65003'
+            router_id: 1.1.1.1
+            timers:
+                bgp:
+                    holdtime: 90
+                    keepalive: 30
+    device_info:
```

Comment:
device_info should be part of all_gathered_resources, is it indentation problem or structure problem ?